libr/include/r_search.h

```diff
@@ -16,7 +16,6 @@ enum {
 	R_SEARCH_ESIL,
 	R_SEARCH_KEYWORD,
 	R_SEARCH_REGEXP,
-	R_SEARCH_PATTERN,
```

Comment:
that should route the call to `r_search_pattern`, the problem is that this 'mode' doesnt follows the same callback model. so it cant reuse the searchupdate loop. will be good to get this fixed too